calibration/experiments/gcm_driven_scm/run_calibration.jl

```diff
@@ -79,23 +85,33 @@ end
 
 series_names = [ref_paths[i] for i in 1:length(ref_paths)]
 
-# minibatcher
+### define minibatcher
 rfs_minibatcher =
-    EKP.RandomFixedSizeMinibatcher(experiment_config["batch_size"])
+    EKP.FixedMinibatcher(collect(1:experiment_config["batch_size"]))
```

Comment:
So this is just creating 1 batch of `[1:batch_size]`. Is that intentional? Or did you want like 
`[[1:bs], [bs+1:2*bs], ... ]`